yt_dlp/extractor/youtube.py

```diff
@@ -261,6 +261,7 @@
             },
         },
         'INNERTUBE_CONTEXT_CLIENT_NAME': 2,
+        'REQUIRE_PO_TOKEN': True,
```

Comment:
can we also update the comment a few lines above? "mweb used to not require PO token with this UA" or something

Reply:
updated